recipes/fltk/all/conanfile.py

```diff
@@ -17,13 +17,15 @@ class FltkConan(ConanFile):
         "with_gl": [True, False],
         "with_threads": [True, False],
         "with_gdiplus": [True, False],
+        "abi_version": "ANY"
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "with_gl": True,
         "with_threads": True,
         "with_gdiplus": True,
+        "abi_version": "10308"
```

Comment:
I strongly advise to not hardcode this value. abi version should match fltk version by default, here you are hardcoding the abi version of a specific fltk version, it won't scale.

Reply:
I have changed that according to your advise. Thank you!